readme.md

```diff
@@ -1,88 +1,183 @@
+  const visitorId = result.visitorId;
+  console.log(visitorId);
 })();
 ```
 
-## Version policy
+## Pro version
 
-The library doesn't promise same visitor identifier between any versions
-but tries to keep them same as much as reasonable.
+Upgrade to [Pro version](https://fingerprintjs.com) to get 99.5% accuracy of identification.
 
-The documented JS API follows [Semantic Versioning](https://semver.org).
-Using undocumented features is at you own risk.
+<p align="center">
+  <a href="https://fingerprintjs.com">
+    <img src="resources/pro_screenshot.png" alt="Pro screenshot" width="697px" />
+  </a>
+</p>
 
-## Contribution
+| | Open Source version | Pro version |
+|-----|-------|-------|
+| Browser identification | Basic | Advanced |
+| Anonymous user identification | ❌ | ✅ |
+| Bot detection | ❌ | ✅ |
+| Incognito / Private mode detection | ❌ | ✅ |
+| Geolocation | ❌ | ✅ |
+| Security | ❌ | ✅ |
+| Server API | ❌ | ✅ |
+| Webhooks | ❌ | ✅ |
 
-### Development playground
+Pro result example:
 
-```bash
-# Make sure you have Yarn installed
-yarn install
-yarn playground:start # Add '--port 8765' to change the server port
+```js
+{
+  "requestId": "HFMlljrzKEiZmhUNDx7Z",
+  "visitorId": "kHqPGWS1Mj18sZFsP8Wl",
+  "visitorFound": true,
+  "incognito": false,
+  "bot": undefined,
+  "ip": "192.65.67.131",
+  "browserName": "Chrome",
+  "browserVersion": "85.0.4183",
+  "os": "Mac OS X",
+  "osVersion": "10.15.6",
+  "device": "Other",
+  "ipLocation": {/* ... */}
+}
 ```
 
-Then open http://localhost:8080 in a browser.
-It's reloaded every time the source code is changed.
+🍿 [Live demo](https://fingerprintjs.com/demo)
 
-To build the playground distribution code (e.g. to upload to a static server), run:
+⏱ [How to upgrade from Open Source to Pro in 30 seconds]()
 
-```bash
-yarn playground:build
-```
+📗 [FingerprintJS Pro documentation](https://dev.fingerprintjs.com)
+
+## Open version reference
+
+The library is shipped in various formats:
+
+- Plain JS
+    ```html
+    <script
+      async src="https://cdn.jsdelivr.net/npm/@fingerprintjs/fingerprintjs@3/dist/fp.min.js"
+      onload="/* ... */"
+    ></script>
+    ```
+- UMD
+    ```js
+    require(['https://cdn.jsdelivr.net/npm/@fingerprintjs/fingerprintjs@3/dist/fp.umd.min'], FingerprintJS => {/* ... */});
+    ```
+- EcmaScript module
+    ```js
+    import * as FingerprintJS from '@fingerprintjs/fingerprintjs';
+    ```
+- CommonJS
+    ```js
+    const FingerprintJS = require('@fingerprintjs/fingerprintjs');
+    ```
 
-The result will appear at `playground/dist`.
+API:
 
-### How to build
+- `FingerprintJS.load({ delayFallback?: number }): Promise<Agent>`
 
-To build the distribution files of the library, run:
+    Builds an instance of Agent and waits a delay required for a proper operation.
+    `delayFallback` is duration (ms) of the fallback for browsers that don't support [requestIdleCallback](https://developer.mozilla.org/en-US/docs/Web/API/Window/requestIdleCallback).
```

Comment:
This should emphasize that this parameter is optional and has a good default which is not recommended to change